core/store/src/main/java/uk/gov/gchq/gaffer/store/schema/Schema.java

```diff
@@ -640,6 +640,12 @@ private void validateSharedGroupsAreCompatible(final Map<String, ? extends Schem
                         continue;
                     }
 
+                    // Check to see if either of the properties are a subset of another properties
+                    if (elementDef2.properties.entrySet().containsAll(elementDef1.properties.entrySet()) ||
```

Comment:
To confirm. because your comparing the entryset, the Key-Value paring must both matched to the other properties, correct.

Reply:
Yeah that's right